City-of-Helsinki/drupal-helfi-platform-config
# UHF-2928: Update cookie categories and cookie intro text

How to test:
* Open the cookie information page `/cookie-information-and-settings` for all translations.
* Get the changes: `composer require "drupal/helfi_platform_config:dev-UHF-2928-cookie-list-updates"`
* Using `make shell`, run:
  * `drush updb`
  * `drush locale-import fi /app/public/modules/contrib/helfi_platform_config/helfi_features/helfi_gdpr_compliance/translations/fi.po`
  * `drush locale-import sv /app/public/modules/contrib/helfi_platform_config/helfi_features/helfi_gdpr_compliance/translations/sv.po`
* Check that the cookie information page is updated, e.g. the  "Preference cookies" is now "Preference" and so on.
